deployments/cardano-prod.nix

```diff
@@ -6,7 +6,13 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["prod"];
+    services.dd-agent.tags = ["env:production"];
+
+    # DEVOPS-64: disable log bursting
+    services.journald.rateLimitBurst = 0;
```

Comment:
Currently, logs are trimmed, so we could be missing important information in log events. This helps us collect all logs.